sbt-version-policy/src/test/scala/sbtversionpolicy/DependencyCheckReportTest.scala

```diff
@@ -32,8 +32,11 @@ object DependencyCheckReportTest extends BasicTestSuite {
       isCompatible("1.0.1",     "1.0.0")
       isBreaking  ("1.1.0",     "1.0.0")
       isBreaking  ("2.0.0",     "1.0.0")
-      isBreaking  ("1.0.0",     "1.0.0-RC1")
+      isBreaking  ("1.1.0",     "1.0.0-RC1")
+      isBreaking  ("2.0.0",     "1.0.0-RC1")
       isCompatible("1.0.0-RC1", "1.0.0-RC1")
+      isCompatible("1.0.0",     "1.0.0-RC1")
```

Comment:
This is inconsistent with what `versions` does, see https://scastie.scala-lang.org/LQVPcpnLSuqRzz9z8J4jQA.

We should ignore the suffix of the current version only (as in https://scastie.scala-lang.org/miZPzsvfS7mrB6FGoyMVEQ).

So, we should have the following assertions:

~~~ scala
isBreaking("1.0.0", "1.0.0-RC1")
isBreaking("1.0.1", "1.0.0-RC1")
isBreaking("1.1.0", "1.0.0-RC1")
isBreaking("2.0.0", "1.0.0-RC1")
isCompatible("1.0.0-RC1", "1.0.0-RC1")
isCompatible("1.1.0-RC1", "1.0.0")
isCompatible("1.2.1-SNAPSHOT", "1.2.0")
~~~